### PR TITLE
lychee: New port

### PR DESCRIPTION
**Pull request by judaew**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.4 21F79 x86_64
Command Line Tools 13.4.0.0.1.1651278267

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: devel/lychee/Portfile [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+github.setup        lycheeverse lychee 0.10.1 v
+github.tarball_from archive
+revision            0
+
+categories          devel
+maintainers         {judaew @judaew} openmaintainer
+license             Apache-2
+
+description         A fast, async, stream-based link checker written in Rust.
+long_description    {*}${destroot} \
```
Comment (herbygillot) [inline devel/lychee/Portfile:16]: I think you meant `{*}${description}`

**Comment by herbygillot** [comment]
Hey @judaew, this looks good.  Just one comment.

**Comment by judaew** [comment]
Thank you @herbygillot 🌟

**Comment by herbygillot** [comment]
Great submission @judaew, thank you.

**Pull request merged**